app/src/main/java/com/dessalines/thumbkey/ui/theme/Color.kt

```diff
@@ -1203,3 +1203,39 @@ fun highContrastColorful(): Pair<ColorScheme, ColorScheme> {
         )
     return Pair(light, dark)
 }
+
+fun matrix(): Pair<ColorScheme, ColorScheme> {
+    val dark =
+        darkColorScheme(
```

Comment:
This clearly wasn't generated like the others using the m3 generator. I'd use that one instead.

Reply:
Can you give more details about what aspects you care about regarding generated themes?

Reply:
I.e. is it code formatting? I simplified it to this because I was hand tweaking it to look right, happy to balloon it if you want it to look like the others

Reply:
I guess its fine, I just hope it doesn't become standard. In the future we might use some of those other vars that come along with the m3 generation, and it stuck out that yours looked different than all the other generated themes.

Reply:
I see, thanks. Any plans in your head regarding other vars? Or just trying to keep options open?

Reply:
Ya just keeping options open. I def like more monotone or simple themes, but there might be a case later on for additional colors in the m3 palette.